src/utils/checkTestEnvironment.ts

```diff
@@ -0,0 +1,15 @@
+export function checkTestEnvironment() {
+  if (process.env.REACT_APP_IS_TEST_ENV !== 'true') {
+    return;
+  }
+
+  const message =
+    'This is a testing page, do not use it with real KILT coins. Do you want to go to the real DIDsign?';
```

Comment:
It wasn't 100% clear to me what i would have to do to get to the testing version. Others might be smarter but I think we could be a bit more explicit here.

```suggestion
    'This is a testing page, do not use it with real KILT coins. T visit the official DIDsign press OK otherwise cancel.';
```

Reply:
As discussed, this is intentional to lower the chances of common users staying on this page. We anticipate developers to be able to figure it out in the end.